embedding-calculator/src/docs/find_faces_post.yml

```diff
@@ -2,7 +2,7 @@ tags:
   - Core
 summary: 'Find faces in the given image and return their bounding boxes.'
 description: 'Returns bounding boxes of detected faces on the image.'
-operationId: scanFacesPost
+operationId: findFacesPost
```

Comment:
Is this swagger file? Then I don't see descriptions of gender and age plugins

Reply:
It's just a fix of wrong name. Swagger description updated in PR #302